frontend-react/src/index.css

```diff
@@ -15,3 +15,7 @@ body,
 #root {
   height: 100%;
 }
+
+.custom-select .ant-select-item {
+  padding-left: 0;
+}
```

Comment:
do we still need this in the css, since we got rid of 'custom-select'?

Reply:
done!